config/encrypt.go

```diff
@@ -10,23 +10,32 @@ import (
 	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/serviceutil"
 )
 
-func Encrypt(cliConnection plugin.CliConnection, configServerInstanceName string, plainText string, authenticatedClient httpclient.AuthenticatedClient) (string, error) {
-	return EncryptWithResolver(cliConnection, configServerInstanceName, plainText, authenticatedClient, serviceutil.ServiceInstanceURL)
+func Encrypt(cliConnection plugin.CliConnection, configServerInstanceName string, plainText string, fileToEncrypt string, authenticatedClient httpclient.AuthenticatedClient) (string, error) {
+	return EncryptWithResolver(cliConnection, configServerInstanceName, plainText, fileToEncrypt, authenticatedClient, serviceutil.ServiceInstanceURL)
 }
 
-func EncryptWithResolver(cliConnection plugin.CliConnection, configServerInstanceName string, plainText string, authenticatedClient httpclient.AuthenticatedClient,
+func EncryptWithResolver(cliConnection plugin.CliConnection, configServerInstanceName string, plainText string, fileToEncrypt string, authenticatedClient httpclient.AuthenticatedClient,
```

Comment:
Not sure about passing both `plainText` and `fileToEncrypt` when it would have been trivial to read the file in the caller and pass just `plainText` in either case.

Reply:
Yeah, I spent a lot of time thinking about that. Would you move the file-reading into `Encrypt`? I originally had it in `plugin.go` but felt that it really didn't belong there, so apparently I overcorrected.

Reply:
I think it's best to move it to `Encrypt`:
* you could, at least in principle, unit test this,
* it will avoid global clutter in `plugin.go`.

Reply:
I gave this a try on Friday and the hardest part was testing it...because I needed to send a fake resolver to `EncryptWithResolver` to test or else it would try to use a real resolver. The tests we have now call `EncryptWithResolver` directly, so this isn't a problem. But if I move the file loading logic up into `Encrypt`, then the tests would need to call `Encrypt` and I'd be faced with how to get a fake resolver in play.

Admittedly, by that time it was later in the day and I was not thinking as clearly, so I intend to give it another shot today. If you have any thoughts that might help, I'd be keen on hearing them.